Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper.json

```diff
@@ -1,101 +1,276 @@
-					"simple": "behaviors.scenario"
-				},
-				"Behaviour Tactic": {
-					"complex": null,
-					"simple": "behaviors.tactic"
-				},
-				"Device External IP": {
-					"complex": null,
-					"simple": "device.external_ip"
-				},
-				"Device Local IP": {
-					"complex": null,
-					"simple": "device.local_ip"
-				},
-				"Device Model": {
-					"complex": null,
-					"simple": "device.platform_name"
-				},
-				"Device Name": {
-					"complex": null,
-					"simple": "device.hostname"
-				},
-				"File Name": {
-					"complex": null,
-					"simple": "behaviors.filename"
-				},
-				"MAC Address": {
-					"complex": null,
-					"simple": "device.mac_address"
+    "description": "CrowdStrike Falcon Mapper for incidents and detections",
+    "feed": false,
+    "id": "CrowdStrike Falcon-Mapper",
+    "mapping": {
+        "CrowdStrike Falcon Detection": {
+            "dontMapEventToLabels": true,
+            "internalMapping": {
+                "Account Name": {
+                    "simple": "behaviors.user_name"
+                },
+                "Alert ID": {
+                    "simple": "detection_id"
+                },
+                "Assigned User": {
+                    "simple": "assigned_to_uid"
+                },
+                "Behaviour Objective": {
+                    "simple": "behaviors.objective"
+                },
+                "Behaviour Scenario": {
+                    "simple": "behaviors.scenario"
+                },
+                "Behaviour Tactic": {
+                    "simple": "behaviors.tactic"
+                },
+                "CMD line": {
+                    "simple": "behaviors.cmdline"
+                },
+                "Cloud Instance ID": {
+                    "simple": "device.instance_id"
+                },
+                "Cloud Service": {
+                    "simple": "device.service_provider"
+                },
+                "Description": {
+                    "simple": "behaviors.description"
+                },
+                "Detected User": {
+                    "simple": "behaviors.user_name"
+                },
+                "Device External IP": {
+                    "simple": "device.external_ip"
+                },
+                "Device Local IP": {
+                    "simple": "device.local_ip"
+                },
+                "Device Model": {
+                    "simple": "device.platform_name"
+                },
+                "Device Name": {
+                    "simple": "device.hostname"
+                },
+                "Agent Version": {
+					"simple": "device.agent_version"
 				},
-				"MD5": {
+                "File MD5": {
+                    "simple": "behaviors.md5"
+                },
+                "MD5": {
 					"complex": null,
 					"simple": "behaviors.md5"
 				},
-				"OS": {
-					"complex": null,
-					"simple": "device.os_version"
-				},
-				"SHA256": {
+                "SHA256": {
```

Comment:
Better use the field **File SHA256**
As its multi select just like you used **File MD5**

Reply:
As discussed - leaving that as it is